static/js/containers/auth/LoginPasswordPage_test.js

```diff
@@ -64,20 +67,16 @@ describe("LoginPasswordPage", () => {
   ].forEach(([extraDataName, extraDataImg, email, descriptor]) => {
     it(`should render the page with correct messages when extra user data ${descriptor}`, async () => {
       const expectedProfileInfo = R.none(R.isNil)([extraDataName, extraDataImg])
```

Comment:
Can be `R.none(R.isNil, [extraDataName, extraDataImg])` instead, which I think is a clearer read.